PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -74,53 +72,63 @@ PATMuonHeavyObjectCache::PATMuonHeavyObjectCache(const edm::ParameterSet& iConfi
-  embedTrack_ = iConfig.getParameter<bool>( "embedTrack" );
-  embedCombinedMuon_ = iConfig.getParameter<bool>( "embedCombinedMuon"   );
-  embedStandAloneMuon_ = iConfig.getParameter<bool>( "embedStandAloneMuon" );
+  embedBestTrack_ = iConfig.getParameter<bool>("embedMuonBestTrack");
+  embedTunePBestTrack_ = iConfig.getParameter<bool>("embedTunePMuonBestTrack");
+  forceEmbedBestTrack_ = iConfig.getParameter<bool>("forceBestTrackEmbedding");
+  embedTrack_ = iConfig.getParameter<bool>("embedTrack");
+  embedCombinedMuon_ = iConfig.getParameter<bool>("embedCombinedMuon");
+  embedStandAloneMuon_ = iConfig.getParameter<bool>("embedStandAloneMuon");
   // embedding of muon MET correction information
-  embedCaloMETMuonCorrs_ = iConfig.getParameter<bool>("embedCaloMETMuonCorrs" );
-  embedTcMETMuonCorrs_ = iConfig.getParameter<bool>("embedTcMETMuonCorrs"   );
-  caloMETMuonCorrsToken_ = mayConsume<edm::ValueMap<reco::MuonMETCorrectionData> >(iConfig.getParameter<edm::InputTag>("caloMETMuonCorrs" ));
-  tcMETMuonCorrsToken_ = mayConsume<edm::ValueMap<reco::MuonMETCorrectionData> >(iConfig.getParameter<edm::InputTag>("tcMETMuonCorrs"   ));
+  embedCaloMETMuonCorrs_ = iConfig.getParameter<bool>("embedCaloMETMuonCorrs");
+  embedTcMETMuonCorrs_ = iConfig.getParameter<bool>("embedTcMETMuonCorrs");
+  caloMETMuonCorrsToken_ =
+      mayConsume<edm::ValueMap<reco::MuonMETCorrectionData>>(iConfig.getParameter<edm::InputTag>("caloMETMuonCorrs"));
+  tcMETMuonCorrsToken_ =
+      mayConsume<edm::ValueMap<reco::MuonMETCorrectionData>>(iConfig.getParameter<edm::InputTag>("tcMETMuonCorrs"));
   // pflow specific configurables
-  useParticleFlow_ = iConfig.getParameter<bool>( "useParticleFlow" );
-  embedPFCandidate_ = iConfig.getParameter<bool>( "embedPFCandidate" );
-  pfMuonToken_ = mayConsume<reco::PFCandidateCollection>(iConfig.getParameter<edm::InputTag>( "pfMuonSource" ));
-  embedPfEcalEnergy_ = iConfig.getParameter<bool>( "embedPfEcalEnergy" );
+  useParticleFlow_ = iConfig.getParameter<bool>("useParticleFlow");
+  embedPFCandidate_ = iConfig.getParameter<bool>("embedPFCandidate");
+  pfMuonToken_ = mayConsume<reco::PFCandidateCollection>(iConfig.getParameter<edm::InputTag>("pfMuonSource"));
+  embedPfEcalEnergy_ = iConfig.getParameter<bool>("embedPfEcalEnergy");
   // embedding of tracks from TeV refit
-  embedPickyMuon_ = iConfig.getParameter<bool>( "embedPickyMuon" );
-  embedTpfmsMuon_ = iConfig.getParameter<bool>( "embedTpfmsMuon" );
-  embedDytMuon_ = iConfig.getParameter<bool>( "embedDytMuon" );
+  embedPickyMuon_ = iConfig.getParameter<bool>("embedPickyMuon");
+  embedTpfmsMuon_ = iConfig.getParameter<bool>("embedTpfmsMuon");
+  embedDytMuon_ = iConfig.getParameter<bool>("embedDytMuon");
+  // embedding of inverse beta variable information
+  muonTimeExtraToken_ =
+      consumes<edm::ValueMap<reco::MuonTimeExtra>>(iConfig.getParameter<edm::InputTag>("sourceMuonTimeExtra"));
```

Comment:
please bring this and other parts of this PR in line with the master version
```diff
-  muonTimeExtraToken_ =
-      consumes<edm::ValueMap<reco::MuonTimeExtra>>(iConfig.getParameter<edm::InputTag>("sourceMuonTimeExtra"));
+  addInverseBeta_ = iConfig.getParameter<bool>("addInverseBeta");
+  if (addInverseBeta_) {
+    muonTimeExtraToken_ =
+        consumes<edm::ValueMap<reco::MuonTimeExtra>>(iConfig.getParameter<edm::InputTag>("sourceMuonTimeExtra"));
+  }
```
either the ```addInverseBeta_``` flag should be dropped in the master or re-added here